src/dynamo.js

```diff
@@ -69,7 +69,10 @@ export class Dynamo {
       TableName: tableGames
     }
     const games = await client.scan(params).catch((e) => console.error(e))
-    const regularGames = games.Items.map((game) => unmarshall(game))
+    const regularGames = games?.Items?.map((game) => unmarshall(game))
+
+    if (!regularGames) return null
```

Comment:
If we return null here,
```js
    const startedGames = await dynamo.getStartedGames(playerId)
    const gamesId = startedGames.map(({ id: gameId }) => {
      return { gameId }
    })
```
When the method is called and then its result mapped, an error would be thrown. What can we return here to mitigate this?